embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -272,23 +191,23 @@ internal sealed class SchemaType(
         override val schemaAttributes = mapOf(
             "duration" to networkCapturedCall.duration.toString(),
             "end-time" to networkCapturedCall.endTime.toString(),
-            "http-method" to networkCapturedCall.httpMethod,
-            "matched-url" to networkCapturedCall.matchedUrl,
+            HttpAttributes.HTTP_REQUEST_METHOD.key to networkCapturedCall.httpMethod,
+            UrlAttributes.URL_FULL.key to networkCapturedCall.matchedUrl,
             "network-id" to networkCapturedCall.networkId,
             "request-body" to networkCapturedCall.requestBody,
-            "request-body-size" to networkCapturedCall.requestBodySize.toString(),
+            HttpIncubatingAttributes.HTTP_REQUEST_BODY_SIZE.key to networkCapturedCall.requestBodySize.toString(),
             "request-query" to networkCapturedCall.requestQuery,
-            "request-query-headers" to networkCapturedCall.requestQueryHeaders.toString(),
+            "http.request.header" to networkCapturedCall.requestQueryHeaders.toString(),
             "request-size" to networkCapturedCall.requestSize.toString(),
             "response-body" to networkCapturedCall.responseBody,
-            "response-body-size" to networkCapturedCall.responseBodySize.toString(),
-            "response-headers" to networkCapturedCall.responseHeaders.toString(),
+            HttpIncubatingAttributes.HTTP_RESPONSE_BODY_SIZE.key to networkCapturedCall.responseBodySize.toString(),
+            "http.response.header" to networkCapturedCall.responseHeaders.toString(),
```

Comment:
Are there no conventions for headers?

Reply:
HttpAttributes.HTTP_REQUEST_HEADER is a stringArrayKeyTemplate, so we can't get the key. 

The docs say this: 

> The attribute value MUST consist of either multiple header values as an array of strings or a single-item array containing a possibly comma-concatenated string, depending on the way the HTTP library provides access to headers.

We are doing the second option, but there is no way of getting the key, at least with the stringArrayKeyTemplate class.

Reply:
Got it. Cool.